scripts/examples.js

```diff
@@ -8,8 +8,8 @@ globSync('./examples/*/').forEach((exampleDir) => {
         cwd: exampleDir,
         stdio: 'inherit',
     };
-
-    const result = spawnSync(command, args, opts);
+    let envCommand=command+ (/^win/.test(process.platform) ? '.cmd' : '');
```

Comment:
I'm also talking to the cross-env guys about why the nested example fail on Windows.  It's not clear whether cross-env or cross-spawn needs correcting, but the fault lies there, not here.